### PR TITLE
Fix invalid configuration in /etc/pam.d/common-password

### PR DESCRIPTION
**Pull request by lihkin213**
This is the same as PR https://github.com/blueboxgroup/ursula/pull/1867 that I am going to close out

### CODE REVIEW

File: roles/common/tasks/password-policy.yml [hunk 1]
```diff
@@ -1,8 +1,16 @@
----
+--
```
Comment (j2sol) [inline roles/common/tasks/password-policy.yml]: Hah, this went from 4 dashes, to 2. I think the standard is 3, but this may work anyway.

**Comment by lihkin213** [comment]
Fixed it.

File: roles/common/tasks/password-policy.yml (hunk 1 above)
Comment (lihkin213) [inline roles/common/tasks/password-policy.yml]: heh. one too many taken out. re-pushed

**Comment by j2sol** [comment]
retest

**Comment by lihkin213** [comment]
Can we please retest this. Looks like the CI aborted this build

**Comment by j2sol** [comment]
Pleaser rebase on master to pick up CI fixes.

**Pull request closed without merging**